theano/tensor/tests/test_basic.py

```diff
@@ -1735,6 +1737,8 @@ def test_py_c_match():
     expected_chi2sf = []
     expected_j0 = []
     expected_j1 = []
+    expected_i0 = []
+    expected_i1 = []
```

Comment:
You need to add the tests it self. Look in this file at J1Tester and J1InplaceTester for an example.